apps/meteor/app/api/server/v1/federation.ts

```diff
@@ -22,3 +22,21 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'federation/configuration.verify',
```

Comment:
Since the settings page uses the meteor method to do its work, are we using this endpoint? If not, we can remove, wdyt?

Reply:
iirc i left it in case others want to know externally.

Reply:
like mobile for any reason